### DISCUSSION TITLE
Can I run the test on just one of the specified browsers via cli?

### DISCUSSION THREAD

**Discussion by fregante** [webdriverio/webdriverio#7353] [Q&A] [answered]
I have this in my config:

```js
    capabilities: [{
        browserName: 'chrome',
    },{
        browserName: 'firefox',
    }],
```

Can I specify which browser to pick via CLI? I want to keep the configuration in the file but usually I only run the test on one driver while working.

**Comment by erwinheitzman** [comment, marked as answer]
The easiest solution is to simply create two separate configs (you can also create a base config to extend from).

**Comment by christian-bromann** [comment]
Another way is read the CLI args within your config file and apply the browser correct browser based on that value.
